src/main/java/com/mixpanel/android/util/MixpanelServerCallback.java

```diff
@@ -0,0 +1,9 @@
+package com.mixpanel.android.util;
+
+import java.util.Map;
+
+public interface MixpanelServerCallback {
```

Comment:
Can we split this into 2
1. `ProxyServerRequestInterceptor` : this will intercept the request and provide the additional custom headers
2. `ProxyResponseCallback` : this will provide a callback for the proxy response